### PR TITLE
2x Mods for V2 Sept2021

### PR DESCRIPTION
**Pull request by khay8485**
* [X] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [X] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [X] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [X] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [X] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [X] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [X] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [X] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [X] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
Two Mods are added.
1. I built a V2 in 200mm², so skirts needed adjustment. For Side skirts, I decided to change the middle part, and fitted it to work with 4020 fans.
2. I wanted the Switch on the side of the printer and found a mod for this. I used the free space on the Plug Panel to add more Keystone Inserts. 
#### How was it tested? 
Both are printed on my V2 and mounted to it. See pics.
#### Screenshots (if appropriate)
Pictures are in the respective folders.

### CODE REVIEW

File: printer_mods/khay/V2_3x_Keystone_Plugpanel/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Adding an unnecessary amount of Keystone Holders to the Plug_Panel component. 
+The part works with "TycoElectronics - 10EGG1-2 (Normal) or 10EHG1-2 (Medical) Filtered Power Inlet" or equal Inlets and needs moving the switch to another skirt 
+like with @tayto-chip´s mod.
+
+![Plug_Panel](Img/Plug_Panel_CAD.png)
+
+![Plug_Panel](Img/DSC_3253.JPG)
+
+tayto-chip´s "skirt_switch_mod":
```
Comment (KiloQubit) [inline printer_mods/khay/V2_3x_Keystone_Plugpanel/README.md:9]: Can you use a relative link?  So these two lines become

    tayto-chip's [Skirt Switch Mod](../../tayto-chip/skirt_switch_mod)

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mods!

Can you make this chamfer go all the way to the vertical wall to avoid an unsupported overhang?
Screen Shot 2021-10-02 at 14 26 32

Please remove the space from `Plug Panel.stl` filename

Also please see README changes below.

File: printer_mods/khay/V2_3x_Keystone_Plugpanel/README.md (hunk 1 above)
Comment (khay8485) [reply printer_mods/khay/V2_3x_Keystone_Plugpanel/README.md]: Hello KiloQubit, updated filename and README for plug_panel mod. 

That chamfer on the 4020 Skirt is the same as in the original V2.4 Skirts, I left that untouched to save me some trouble remodeling. If you want me to change it, pls just comment and I will give that some more rounding off.

Comment (KiloQubit) [reply printer_mods/khay/V2_3x_Keystone_Plugpanel/README.md]: I didn't realize the stock skirts were like that, this is fine then.

**Review by KiloQubit — APPROVED** [review]
Thanks for the updates!

**Pull request merged**